library/src/G4CMPCrystalGroup.cc

```diff
@@ -10,7 +10,8 @@
 //
 // 20170728  Change function args "alpha, beta, gamma" to "al, bt, gm" (-Wshadow)
 // 20240426  S. Zatschler -- Add explicit fallthrough statements to switch cases
-// 20240924  I. Hernandez -- Add an if statement to include a rhombohedral Crystal group and filling more explicitly the function G4CMPCrystalGroup::FillOrthorhombic()
+// 20241014  I. Hernandez -- Add an if statement to include a rhombohedral Crystal group.
```

Comment:
This line appears to have a whole lot of blanks (space characters, ASCII 0x20) following the period.  Please remove all of them.